impeller/entity/shaders/sweep_gradient_fill.frag

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <impeller/texture.glsl>
+
+uniform GradientInfo {
+  vec2 center;
+  float bias;
+  float scale;
+  vec4 start_color;
+  vec4 end_color;
+  float tile_mode;
+} gradient_info;
+
+in vec2 interpolated_vertices;
+
+out vec4 frag_color;
+
+void main() {
+  vec2 coord = interpolated_vertices - gradient_info.center;
+  float angle = atan(-coord.y, -coord.x);
+
+  // 0.1591549430918 is 1/(2*pi), used since atan returns values [-pi, pi]
+  float t = (angle * 0.1591549430918 + 0.5 + gradient_info.bias) * gradient_info.scale;
```

Comment:
This magic number looks like it would make a good addition to `impeller/constants.glsl` in the shader library. Maybe call it `k1Over2Pi` to keep with the convention in `geometry/constants.h`?